driver/build.gradle

```diff
@@ -43,7 +43,11 @@ clirr {
 
 tasks.withType(Checkstyle) {
     // needed so the Javadoc checks can find the code in other modules
-    classpath = files(project(':driver-core').sourceSets.main.output, sourceSets.main.output);
+    classpath = files(project(':driver-core').sourceSets.main.output, sourceSets.main.output)
+    reports {
```

Comment:
Why is this reports section necessary, and if it is, why don't we need it for bson and driver-core?